src/main/java/org/tikv/common/TiSession.java

```diff
@@ -103,6 +120,22 @@ public TiSession(TiConfiguration conf) {
     warmUp();
     this.circuitBreaker = new CircuitBreakerImpl(conf);
     logger.info("TiSession initialized in " + conf.getKvMode() + " mode");
+    logger.info("Welcome to TiKV Java Client {}", getVersionInfo());
+  }
+
+  private VersionInfo getVersionInfo() {
+    VersionInfo info;
+    try {
+      final Properties properties = new Properties();
+      properties.load(this.getClass().getClassLoader().getResourceAsStream("git.properties"));
+      String version = properties.getProperty("git.build.version");
+      String commitHash = properties.getProperty("git.commit.id.full");
+      info = new VersionInfo(version, commitHash);
+    } catch (Exception e) {
+      logger.info("Fail to read package info: " + e.getMessage());
```

Comment:
```suggestion
      logger.error("Fail to read package info: " + e.getMessage());
```

Reply:
conflicted with: https://github.com/tikv/client-java/pull/408#discussion_r770188489. Personally speaking, I think the error log level could help us discover some building issues.